conf/dkfz.config

```diff
@@ -0,0 +1,30 @@
+
+params {
+    config_profile_contact     = 'Kübra Narcı kuebra.narci@dkfz-heidelberg.de'
+    config_profile_name        = 'DKFZ cluster'
+
+    max_cpus   = 30
+    max_memory = '250.GB'
+    max_time   = '48.h'
+}
+
+
+singularity {
+  enabled = true
+  cacheDir = "/omics/groups/OE0246/shared/nf_cache_dir"
+  autoMounts = true
+  runOptions = "-B /omics/groups -B /omics/odcf/analysis -B /omics/odcf/project -B /omics/odcf/reference_data"
```

Comment:
Usually not necessary, if these are mounted and accessible to your user they should normally be auto-mounted for you.

Reply:
should I remove them?

Reply:
Ideally check first if you can run without them, then remove if that holds true. I suspect if your Singularity setup is similar to what most people use, that this is the case and you then could remove them.

Reply:
ok, I updated the config and made it more general. cacheDir will be defined in env options.